### DISCUSSION TITLE
Code of conduct violation.

### DISCUSSION THREAD

**Discussion by solomancode** [avajs/ava#3056] [General]
Closing PR #3055  is a clear violation to the contributor's code of conduct.

**Statement 1, 4 and 5, Section 2 of the Contributor's Covenant Code of Conduct**
- Using welcoming and **inclusive** language
- **Gracefully** accepting constructive criticism
- Showing **empathy** towards other community member

**Comment by novemberborn** [comment]
I explained why the current banner is in place. I explained why we won't consider adding additional banners.

I asked you to refrain from pursuing this further.

Please do so now.

**Comment by solomancode** [comment]
Ok, I will. Since you've only asked me now - The other one was an order since there was no room for a discussion. To be honest I never thought you'd place the banner anyway. I'm very disappointed, I don't remember how many times I mentioned AVA as my favourite testing framework. You didn't have to place the banner in Readme, That'd bring a lot of trouble, something that I already know. You could've shown some empathy towards the people of Palestine as well as part of being a human - Life is a test, and you can't build a great character without passing all the tests, did you know that.
